lib/plugins/aws/deploy/compile/functions/iam-policy-lambda-vpc-template.json

```diff
@@ -0,0 +1,30 @@
+{
+  "IamPolicyLambdaVpc": {
+    "Type": "AWS::IAM::ManagedPolicy",
+    "Properties": {
+      "PolicyName": "",
+      "PolicyDocument": {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "logs:CreateLogGroup",
```

Comment:
The log actions aren't necessary as those are already in the main IAM Policy we create. We only need the ec2:* actions in here